src/Project/SmootherAPI.jl

```diff
@@ -0,0 +1,107 @@
+
+"""
+    addSpringSmoother!(status::String, type::String, nIterations::Int)
+
+Status is either `ON` or `OFF`
+Type is either `LinearSpring` or `LinearAndCrossbarSpring`
+"""
+function addSpringSmoother!(proj::Project,status::String = "ON",
+                            type::String = "LinearAndCrossbarSpring",
+                            nIterations::Int = 25)
```

Comment:
```suggestion
function addSpringSmoother!(proj::Project, type::String = "LinearAndCrossbarSpring",
                            nIterations::Int = 25, status::String = "ON")
```
Very minor suggestion (ignore if you'd like) but an argument setting the status as "OFF" seems like it would be used much less often than specifying the spring `type`. Reordering arguments would make it so users can specify a spring smoother without needing to add extra args., e.g., 
```julia
addSpringSmoother!(proj, "LinearSpring") # vs
addSpringSmoother!(proj, "ON", "LinearSpring")
```

Reply:
Turning off the springs can be helpful when meshing a particularly tricky model. If your model curves have sharp corners / cusps / other such difficult features the spring smoother can exacerbate bad elements. So once you adjust the model and background grid combo to work then you can turn the springs back on and the mesh becomes significantly better.

That being said, I think reordering the arguments of the `addSpringSmoother` would be fine

Reply:
If that's true, let's leave the order as-is then. Sounds like it might be more common to turn a spring off then to change its spring type while meshing a new domain.